### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by cyfyifanchen**
* Consistency change
* restructure modals and extension part
* Adding 11labs as option in the config part 
* Adding .next to gitignore

### CODE REVIEW

**Review by plutoless — CHANGES_REQUESTED** [review]
the concept of interchangeable is not clear

**Review by sunshinexcode — APPROVED** [review]
LGTM

**Comment by cyfyifanchen** [comment]
It means it can be changed to the other extension. I think I do have a little paragraph describes it just right above the table. 

Another word choice is switchable, I ended up choose interchangeable, which I think is little bit better(I also asked GPT about it, he thinks the same). I am open to suggestions on any other better word choices.

**Comment by plutoless** [comment]
> It means it can be changed to the other extension. I think I do have a little paragraph describes it just right above the table.
> 
> Another word choice is switchable, I ended up choose interchangeable, which I think is little bit better(I also asked GPT about it, he thinks the same). I am open to suggestions on any other better word choices.

by far we do not yet have docs to describe how to switch extensions (you will need to manipulate manifest.json). the purpose of that section is to help developer understand the default set up, so emphasizing interchangeable is not very useful, and too early for a beginner.

i would suggest we only describe the default setup here. if needed, you can add an additional section abt how to switch extensions. also i would recommend we do this after our graph designer is ready.

**Comment by cyfyifanchen** [comment]
> > It means it can be changed to the other extension. I think I do have a little paragraph describes it just right above the table.
> > Another word choice is switchable, I ended up choose interchangeable, which I think is little bit better(I also asked GPT about it, he thinks the same). I am open to suggestions on any other better word choices.
> 
> by far we do not yet have docs to describe how to switch extensions (you will need to manipulate manifest.json). the purpose of that section is to help developer understand the default set up, so emphasizing interchangeable is not very useful, and too early for a beginner.
> 
> i would suggest we only describe the default setup here. if needed, you can add an additional section abt how to switch extensions. also i would recommend we do this after our graph designer is ready.

Updated.

File: README.md [hunk 1]
```diff
@@ -73,39 +75,80 @@ You will need to uncheck "Use Rosetta for x86_64/amd64 emulation on apple silico
...
+  <td align="center">openai_chatgpt</td>
+    <!-- <td align="center">✅</td> -->
+    <td align="center">LLM</td>
+    <td align="center">Large Language Modal</td>
+  </tr>
+  <tr>
+  <td align="center">elevenlabs_tts</td>
+    <!-- <td align="center">✅</td> -->
+    <td align="center">Text-to-speech</td>
+    <td align="center">Convert text to audio</td>
```
Comment (plutoless) [inline README.md:119]: Elevenlabs Text-to-speech extension?

File: README.md [hunk 2]
```diff
@@ -73,39 +75,80 @@ You will need to uncheck "Use Rosetta for x86_64/amd64 emulation on apple silico
...
+  </tr>
+  <tr>
+  <td align="center">azure_tts</td>
+    <!-- <td align="center">✅</td> -->
+    <td align="center">Text-to-speech</td>
+    <td align="center">Convert text to audio</td>
```
Comment (plutoless) [inline README.md:125]: same. maybe add vendor info otherwise the description column looks repetive.

**Review by plutoless — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (cyfyifanchen) [reply README.md:119]: That's right.

File: README.md (hunk 2 above)
Comment (cyfyifanchen) [reply README.md]: updated

**Comment by cyfyifanchen** [comment]
image

**Review by plutoless — APPROVED** [review]

**Pull request merged**